source/print.d

```diff
@@ -57,14 +61,27 @@ class Printer {
     string getRowContent(int width, string content) {
         if (content.length > width - 4)
             content = format("%-.*s...", width - 7, content);
-        
+
         return format("%s %-*s %s", vbchar, width - 4, content, vbchar);
     }
 
     string[] printList(List list) {
+
+      if (0 == cmp(this.displayMode, "list")) {
```

Comment:
Could this just be `this.displayMode == "list"`? D is not C, luckily - I think this works here.

[`string.cmp`](https://dlang.org/library/std/string.html) internally seems to reference [`std.algorithm.comparison.cmp`](https://dlang.org/library/std/algorithm/comparison/cmp.html) which just seems strange to use as an equality check. Unless you're using C, in which case it makes perfect sense, because it's C.